extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -261,19 +278,7 @@ export class JupyterServerInstallation {
 	}
 
 	private isPythonRunning(pythonInstallPath: string): Promise<boolean> {
-		let pythonExePath = JupyterServerInstallation.getPythonExePath(pythonInstallPath, this._usingExistingPython);
-		return new Promise<boolean>(resolve => {
-			fs.open(pythonExePath, 'r+', (err, fd) => {
-				if (!err) {
-					fs.close(fd, err => {
-						this.apiWrapper.showErrorMessage(utils.getErrorMessage(err));
-					});
-					resolve(false);
-				} else {
-					resolve(err.code === 'EBUSY' || err.code === 'EPERM');
-				}
-			});
-		});
+		return Promise.resolve(false);
```

Comment:
Disabling this for now, since the current implementation runs into permission issues when using Python in a system directory.

Reply:
Tracking issue for this: https://github.com/microsoft/azuredatastudio/issues/6255